packages/InteractiveProfilingTool-Core.package/IPTMessageTallyWrapper.class/instance/getMemoryStats.st

```diff
@@ -0,0 +1,20 @@
+information extraction
+getMemoryStats
+
+	| memoryEnd oldSpaceEnd youngSpaceEnd |
+	
+	self messageTally gcStats ifNil: [^ nil].
+	
+	oldSpaceEnd			:= self messageTally gcStats at: 1.
+	youngSpaceEnd		:= self messageTally gcStats at: 2.
+	memoryEnd			:= self messageTally gcStats at: 3.
```

Comment:
Please consider moving the gc data accessing logic to seperate functions.

Reply:
Auch in diesem Fall würde die `API`aus meiner Sicht unübersichtlicher werden. Zudem wäre der Code insgesammt schlechter zu lesen.